internal/driver/registry_default.go

```diff
@@ -334,6 +353,18 @@ func (r *RegistryDefault) WriteRouter() http.Handler {
 	return handler
 }
 
+func (r *RegistryDefault) MetricsRouter() http.Handler {
+	n := negroni.New(reqlog.NewMiddlewareFromLogger(r.Logger(), "keto").ExcludePaths(prometheus.MetricsPrometheusPath))
+	router := &x.MetricsRouter{Router: httprouter.New()}
+
+	r.PrometheusManager().RegisterRouter(router.Router)
+	r.MetricsHandler().SetRoutes(router.Router)
+	n.UseHandler(router)
+	n.Use(r.PrometheusManager())
```

Comment:
The router types are there to ensure on a compile level that we don't accidentally register write routes on the read endpoint. For metrics you can just use

```suggestion
	router := httprouter.New()

	r.PrometheusManager().RegisterRouter(router)
	r.MetricsHandler().SetRoutes(router)
	n.UseHandler(router)
	n.Use(r.PrometheusManager())
```